gallery/src/gallery-components/GalleryTiles.tsx

```diff
@@ -124,6 +153,27 @@ export const GalleryExampleTile: FunctionComponent<GalleryExampleTileProps> =
               <h2 className="nx-accordion__header-title">Example Code</h2>
             </NxAccordion.Header>
             {codeExampleElements}
+            <footer className="nx-footer">
```

Comment:
`GalleryExampleTile` is getting pretty long, especially with my changes in RSC-842.  Due to that I think it would be a good idea if the footer and the copy logic were in a separate react component.

Reply:
> `GalleryExampleTile` is getting pretty long, especially with my changes in RSC-842. Due to that I think it would be a good idea if the footer and the copy logic were in a separate react component.

Separate component has been created: 4922038b4e724fd2809aa2cb1858ce8ae89a7ff6